### PR TITLE
Add new columns (configuration path, service environment) to operation

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph/operations/admin.py [hunk 1]
```diff
@@ -64,12 +68,65 @@ def __init__(self, *args, **kwargs):
...
+
+    def get_configuration_path(self, obj):
+        return self._get_related_objects(obj=obj, field='configuration_path')
+    get_configuration_path.allow_tags = True
+    get_configuration_path.short_description = _('configuration path')
+
+    def get_changelist(self, request, **kwargs):
+        return OperationChangeList
+
+    def get_queryset(self, request):
```
Comment (mkurek) [inline src/ralph/operations/admin.py]: is it necessary?

File: src/ralph/operations/admin.py [hunk 2]
```diff
@@ -64,12 +68,65 @@ def __init__(self, *args, **kwargs):
...
+        return self._get_related_objects(obj, 'service_env')
+    get_services.allow_tags = True
+    get_services.short_description = _('services')
+
+    def get_configuration_path(self, obj):
+        return self._get_related_objects(obj=obj, field='configuration_path')
+    get_configuration_path.allow_tags = True
+    get_configuration_path.short_description = _('configuration path')
+
+    def get_changelist(self, request, **kwargs):
```
Comment (mkurek) [inline src/ralph/operations/admin.py]: is is necessary?

Comment (mkurek) [inline src/ralph/operations/admin.py]: is is necessary?

**Review by mkurek — CHANGES_REQUESTED** [review]
please write some test(s) checking that expected number of SQL queries on listing of Operations (> 1)

File: src/ralph/operations/tests/test_admin.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+from ralph.operations.models import OperationType
+from ralph.operations.tests.factories import OperationFactory
+
+from ralph.tests.mixins import ClientMixin
+
+
+class OperationAdminViewTest(ClientMixin, TestCase):
+    def test_operation_changelist_should_run_2_queries(self):
+        OperationFactory.create_batch(5)
```
Comment (mkurek) [inline src/ralph/operations/tests/test_admin.py:12]: unfortunately, `OperationFactory` does not assign service_env and config. path, so this doesn't really test your code :)

**Review by mkurek — CHANGES_REQUESTED** [review]

File: src/ralph/operations/tests/test_admin.py (hunk 3 above)
Comment (ar4s) [reply src/ralph/operations/tests/test_admin.py:12]: done

**Review by mkurek — APPROVED** [review]

**Pull request merged**